### PR TITLE
Feature/map controllers

### PR DESCRIPTION
**Pull request by rorduno**
Added radio btn functionality, clean  whereClauses and remove locationColumn var. Users can now filter by detainment status and date.

### CODE REVIEW

File: js/maps_lib.js [hunk 1]
```diff
@@ -165,6 +165,7 @@
                 'address': address
             }, function (results, status) {
                 if (status === google.maps.GeocoderStatus.OK) {
+                    console.log('getgeoCondition>>>');
```
Comment (eabedrapo) [inline js/maps_lib.js]: Why are you showing this in the console? Is that related to some work in progress?

File: js/maps_lib.js [hunk 2]
```diff
@@ -209,22 +210,37 @@
     };
 
     MapsLib.prototype.doSearch = function () {
+        console.log('doSearch>>>');
```
Comment (eabedrapo) [inline js/maps_lib.js]: Same as in line 168. Will you need this message in the console to debug future code?

File: js/maps_lib.js [hunk 3]
```diff
@@ -209,22 +210,37 @@
...
+        console.log('doSearch whereClause is >>> ' + self);
```
Comment (eabedrapo) [inline js/maps_lib.js]: Same as above.

File: js/maps_lib.js [hunk 4]
```diff
@@ -209,22 +210,37 @@
...
+	        self.whereClause += " AND Detentions='Unknown/unsure'"
+        }
         //-----end of custom filters-----
 
         self.getgeoCondition(address, function (geoCondition) {
             self.whereClause += geoCondition;
             self.submitSearch(self.whereClause, self.map);
         });
-
+		console.log('posting>>>');
```
Comment (eabedrapo) [inline js/maps_lib.js]: Same as above.

**Review by eabedrapo — COMMENTED** [review]
All good, just a few questions on the need of console.log in your changes to maps_lib.js

**Comment by rorduno** [comment]
@eabedrapo removed logs

**Review by eabedrapo — APPROVED** [review]
:+1:

**Comment by guillermokrh** [comment]
Great job @rorduno! Glad you were able to fix the date slider bug!
